src/Views/listings/view.phtml

```diff
@@ -8,31 +8,47 @@ use GroupThr3e\AJExchange\Models\Listing;
 $title = 'listing';
 ?>
 
-<div class="view-listing-grid-container my-5 listing-grid-design mx-auto">
-    <div class="view-listing-main-image listing-grid-design">
+<div class="view-listing-grid-container mx-auto">
+    <div class="view-listing-main-image">
         <img class="mainImage h-100 w-100" src="https://dummyimage.com/200x200/000/ffffff&text=Listings" alt="">
     </div>
     <div class="view-listing-header listing-grid-design">
-        <h6><?= $listing->getUser()->getFullName() ?>, <?= $listing->getUser()->getOffice()->getAddress() ?></h6>
         <div>
-            <h4><?= htmlspecialchars($listing->getListingName()) ?></h4>
+            <h1> <?=htmlspecialchars($listing->getListingName()) ?></h1>
```

Comment:
Remove the whitespace here and at the start of other tags with similar whitespace